documentation/features/set-volume.mdx

```diff
@@ -0,0 +1,13 @@
+# Set Volume
+
+This is a android only API. It allows you to vary the volume level of the remote peer individually for yourself. 
+Suppose a remote peer is too loud compared to others you can reduce their volume level and vice versa.
+Volume can vary from 0(min) to 1(max).
```

Comment:
this will be updated to 0 min to 10 max for both android and ios from next release. Also we should mention 1.0 is default